project_issue_task_sync/models/project_task.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    sync_tasks_issues = fields.Boolean(
+        related=['project_id', 'sync_tasks_issues']
+    )
+    issue_ids = fields.One2many('project.issue', 'task_id', 'Issues')
```

Comment:
this is the inverse of the already existing core field: https://github.com/OCA/OCB/blob/8.0/addons/project_issue/project_issue.py#L252 - in practice, we use this as a 1:1 relation